client/app/src/services/network.service.js

```diff
@@ -326,6 +328,29 @@
       return deferred.promise
     }
 
+    // Returns the BTC value in satoshi
+    function convertToSatoshi(val) {
+        return Number(val).toFixed(8);
+    }
+
+
+    // Updates peer with all currency values relative to the USD price.
+    function updatePeerWithCurrencies(peer, res) {
+        $http.get('https://api.fixer.io/latest?base=USD', { timeout: 2000}).then( function (result) {
+            const USD_PRICE = Number(res.data[0].price_usd)
+            var currencies = ["aud", "brl", "cad", "chf", "cny", "eur", "gbp", "hkd", "idr", "inr", "jpy", "krw", "mxn", "rub"]
+            var prices = {}
+            currencies.forEach(function(currency) {
```

Comment:
There's a shorter way of doing a function (not a massive difference or required, but just an alternative):

```
currencies.forEach(currency => {
    prices[currency] = result.data.rates[currency.toUpperCase()] * USD_PRICE
})
```